README.md

```diff
@@ -24,43 +24,48 @@ Also, includes a Heroku `app.json` and a working Django `production.py` settings
 This is a good starting point for modern Python/JavaScript web projects.
 
 ## Project bootstrap [![CircleCI](https://circleci.com/gh/vintasoftware/django-react-boilerplate.svg?style=svg)](https://circleci.com/gh/vintasoftware/django-react-boilerplate)
+- [ ] Open the command line and go to the directory you want to start your project in.
 - [ ] Start your project using:
 ```
 django-admin startproject theprojectname --extension py,yml,json --name Procfile,README.md,.env.example --template=https://github.com/vintasoftware/django-react-boilerplate/archive/boilerplate-release.zip
 ```
-- [ ] Above: don't forget the `--extension` and `--name` params!
-- [ ] Install pipenv if not installed yet: `pip install pipenv` (maybe you have to run this command as an OS superuser)
-- [ ] Make sure you have Python 3.6 installed
+- [ ] Above: don't forget the `--extension` and `--name` params! (also don't forget to change `theprojectname` to your project's name).
+- [ ] Enter the project's directory through your command line.
+- [ ] Install pipenv if not installed yet: `pip install pipenv` (maybe you'll have to run this command as an OS superuser).
+- [ ] Make sure you have Python 3.6 installed.
 - [ ] `pipenv install --dev`
 - [ ] Activate the newly created virtualenv with `pipenv shell`
 - [ ] `npm update --save`
 - [ ] `npm update --save-dev`
-- [ ] Check for outdated npm dependencies with `npm outdated` and update them
-- [ ] Change the first line of README to the name of the project
-- [ ] Add an email address to the `ADMINS` settings variable
+- [ ] Check for outdated npm dependencies with `npm outdated` and update them.
+- [ ] Change the first line of README to the name of the project.
+- [ ] Add an email address to the `ADMINS` settings variable in `theprojectname/theprojectname/settings/base.py`
 - [ ] Change the `SERVER_EMAIL` to the email address used to send e-mails.
 
 After completing ALL of the above, remove this `Project bootstrap` section from the project README. Then follow `Running` below.
 
 ## Running
 ### Setup
 - On project root, do the following:
-- Create a copy of ``{{project_name}}/settings/local.py.example``:  
-  `cp {{project_name}}/settings/local.py.example {{project_name}}/settings/local.py`
-- Create a copy of ``.env.example``:  
+- Create a copy of ``theprojectname/settings/local.py.example``:  
```

Comment:
I don't think it's a good practice to remove the curly braces. They're sort of a convention (not only curly braces, but also `< >` and similar symbols) to represent data that should be replaced in a string (the same way the `.format()` method works on Python). Adding the explanation in parenthesis could be a good point, but we should keep the braces, IMO.